train.py

```diff
@@ -85,7 +91,7 @@ def total_variation_regularization(x, beta=1):
 feature_s = vgg(Variable(style_b, volatile=True))
 gram_s = [gram_matrix(y) for y in feature_s]
 
-for epoch in range(n_epoch):
+for epoch in range(m_epoch, n_epoch):
```

Comment:
I would interpret n_epoch as n further epochs to train, not the total number.

I can confirm it works, although my trainings didn't get much better after 2 epochs.